src/Main.ml

```diff
@@ -168,6 +173,12 @@ let () =
         Arg.Set print_unknown_externals,
         " Print all the external definitions which are not listed in the \
          builtin functions" );
+      ( "-log",
+        Arg.String add_activated_logger,
+        " Activate debugging log for a given logger designated by its name. \
+         The existing loggers are: {"
+        ^ String.concat ", " (Collections.StringMap.keys !loggers)
+        ^ "}" );
```

Comment:
Thanks for this, I'd wanted it for a while :pray:

Reply:
Yes, it had become really painful ^^